ros2node/ros2node/api/__init__.py

```diff
@@ -20,6 +20,19 @@
 HIDDEN_NODE_PREFIX = '_'
 
 NodeName = namedtuple('NodeName', ('name', 'namespace', 'full_name'))
+TopicInfo = namedtuple('Topic', ('name', 'types'))
+
+
+def parse_node_name(full_node_name):
+    tokens = full_node_name.split('/')
+    if 1 > len(tokens):
```

Comment:
Maybe we should have a guard against cases for empty string or "'/'" instead? I think `split` will always return at least one element, even if it is the empty string.